diff -urp mc-4.7.0.2/lib/vfs/mc-vfs/extfs/iso9660.in mc-4.7.0.2.new/lib/vfs/mc-vfs/extfs/iso9660.in
--- mc-4.7.0.2/lib/vfs/mc-vfs/extfs/iso9660.in	2010-01-29 09:09:27.000000000 +0100
+++ mc-4.7.0.2.new/lib/vfs/mc-vfs/extfs/iso9660.in	2015-10-15 18:59:39.387011295 +0200
@@ -62,7 +62,8 @@ BEGIN {
   sub(" $", "", name)
   ## sub(";[0-9]+$", "", name) ## would break copyout
   # skip . and ..
-  if (name ~ /^\.\.?/) next;
+  if (name == ".") next;
+  if (name == "..") next;
   printf "%s%s%s\n", attr, dir, name
 }' 
 }
