diff -urpN mc-4.7.0.2.orig/doc/man/mc.1.in mc-4.7.0.2/doc/man/mc.1.in
--- mc-4.7.0.2.orig/doc/man/mc.1.in	2010-02-02 10:07:24.000000000 +0100
+++ mc-4.7.0.2/doc/man/mc.1.in	2016-02-10 16:53:14.436384085 +0100
@@ -80,7 +80,8 @@ as an alias to the appropriate shell scr
 .I \-s
 Set alternative mode drawing of frameworks.
 If the section [Lines] is not filled, the symbol for the pseudographics
-frame is a space, otherwise the frame characters are taken from follow params.
+frame is a space, otherwise the frame characters are taken from following
+parameters.
 
 .B You can redefine the following variables:
 .TP
@@ -3610,7 +3611,7 @@ list of extensions of files. Separated b
 .TP
 .I extensions_case
 (make sense only with 'extensions' parameter) make 'extensions'
-rule case sentitive (true) or not (false).
+rule case sensitive (true) or not (false).
 .PP
 `type' key may have values:
 .nf
diff -urpN mc-4.7.0.2.orig/doc/man/mcedit.1.in mc-4.7.0.2/doc/man/mcedit.1.in
--- mc-4.7.0.2.orig/doc/man/mcedit.1.in	2010-01-27 11:53:40.000000000 +0100
+++ mc-4.7.0.2/doc/man/mcedit.1.in	2016-02-10 16:53:14.438384086 +0100
@@ -137,23 +137,23 @@ in the
 menu.
 .SH CODE NAVIGATION
 .B mcedit
-can be used to navigation through code with tags files created by etags
-or ctags commands. If there is no file TAGS code navigation would not work.
-In example, in case of exuberant\-ctags for C language command will be:
+can be used for navigation through code with tags files created by etags
+or ctags commands. If there is no TAGS file code navigation will not work.
+For example, in case of exuberant\-ctags for C language command will be:
 .PP
 ctags \-e \-\-language\-force=C \-R ./
 .PP
-.B Meta\-Enter 
-show list box to select item under cursor (cusor should stand at end of
-word).
-.PP
-.B Meta\-Minus 
-where minus is symbol "\-" go to previous function in navigation list (like a browser
-Back).
+.B Meta\-Enter
+shows list box to select item under cursor (cursor should stand at the end
+of the word).
+.PP
+.B Meta\-Minus
+where minus is symbol "\-" goes to previous function in navigation list
+(like browser's Back button).
 .PP
 .B Meta\-Equal
-where equal is symbol "=" go to next function in navigation list (like a browser
-Forward).
+where equal is symbol "=" goes to next function in navigation list
+(like browser's Forward button).
 .PP
 .SH SYNTAX HIGHLIGHTING
 .B mcedit
@@ -400,7 +400,7 @@ assume a tab spacing of 8. Use
 to simulate a smaller tab spacing.
 .TP
 .I editor_fill_tabs_with_spaces
-Never insert a tab space. Rather insert spaces (ascii 20h) to fill to the
+Never insert a tab character. Rather insert spaces (ascii 32) to fill to the
 desired tab size.
 .TP
 .I editor_return_does_auto_indent
@@ -424,38 +424,38 @@ When editing anywhere else, a normal tab
 .I editor_option_save_mode
 Possible values 0, 1 and 2.  The save mode (see the options menu also)
 allows you to change the method of saving a file.  Quick save (0) saves
-the file by immediately, truncating the disk file to zero length (i.e.
-erasing it) and the writing the editor contents to the file.  This
+the file immediately, truncating the disk file to zero length (i.e.
+erasing it) and then writing the editor contents to the file.  This
 method is fast, but dangerous, since a system error during a file save
 will leave the file only partially written, possibly rendering the data
 irretrievable.  When saving, the safe save (1) option enables creation
 of a temporary file into which the file contents are first written.  In
-the event of an problem, the original file is untouched.  When the
+the event of a problem, the original file is untouched.  When the
 temporary file is successfully written, it is renamed to the name of the
 original file, thus replacing it.  The safest method is create backups
-(2).  Where a backup file is created before any changes are made.  You
+(2): a backup file is created before any changes are made.  You
 can specify your own backup file extension in the dialog.  Note that
 saving twice will replace your backup as well as your original file.
 .TP
 .I editor_word_wrap_line_length
-line length to wrap. 72 default.
+Line length to wrap at. Default is 72.
 .TP
 .I editor_backup_extension
-symbol for add extension to name of backup files. Default "~".
+Symbol to add to name of backup files. Default is "~".
 .TP
 .I editor_line_state
-show state line of editor now it show number of file line (in future it
-can show things like folding, breakpoints, etc.). M\-n toglle this option.
+Show state line of editor. Currently it shows current line number (in the future
+it might show things like folding, breakpoints, etc.). M\-n toggles this option.
 .TP
 .I editor_visible_spaces
-Toggle show visible trailing spaces (TWS), if editor_visible_spaces=1 TWS
-showed as '.'
+Toggle "show visible trailing spaces".  If editor_visible_spaces=1, they are shown
+as '.'
 .TP
 .I editor_visible_tabs
-Toggle show visible tabs, if editor_visible_tabs=1 tabs showed as '<\-\-\-\->'
+Toggle "show visible tabs".  If editor_visible_tabs=1, tabs are shown as '<\-\-\-\->'
 .TP
 .I editor_persistent_selections
-Do not remove block selection after moving the cursor.
+Do not remove block selection after cursor movement.
 .TP
 .I editor_cursor_beyond_eol
 Allow moving cursor beyond the end of line.
@@ -464,7 +464,7 @@ Allow moving cursor beyond the end of li
 enable syntax highlighting.
 .TP
 .I editor_edit_confirm_save
-show confirm dialog on save.
+Show confirmation dialog on save.
 .TP
 .I editor_option_typewriter_wrap
 to be described
